From mfm@muteddisk.com  Fri Oct  2 15:02:19 2009
From: matt mooney <mfm@muteddisk.com>
Date: Wed, 30 Sep 2009 19:25:02 -0700
Subject: Staging: comedi: mite.h: deletion of unused functions
To: Greg KH <greg@kroah.com>
Message-ID: <20091001022502.GD9721@ibook.muteddisk.com>
Content-Disposition: inline


drivers/mite.h: removed declared but undefined functions mite_ll_from_kvmem
and mite_setregs

Signed-off-by: matt mooney <mfm@muteddisk.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/mite.h |    7 -------
 1 file changed, 7 deletions(-)

--- a/drivers/staging/comedi/drivers/mite.h
+++ b/drivers/staging/comedi/drivers/mite.h
@@ -163,13 +163,6 @@ u32 mite_bytes_in_transit(struct mite_ch
 unsigned mite_get_status(struct mite_channel *mite_chan);
 int mite_done(struct mite_channel *mite_chan);
 
-#if 0
-unsigned long mite_ll_from_kvmem(struct mite_struct *mite,
-				 struct comedi_async *async, int len);
-void mite_setregs(struct mite_struct *mite, unsigned long ll_start, int chan,
-		  int dir);
-#endif
-
 void mite_prep_dma(struct mite_channel *mite_chan,
 		   unsigned int num_device_bits, unsigned int num_memory_bits);
 int mite_buf_change(struct mite_dma_descriptor_ring *ring,
